### ISSUE TITLE
is multi-class available in hetero-nn (for FATE from only Pypi)?

### ISSUE THREAD

**Issue by EthanLifeGreat** [FederatedAI/FATE#5500]
I'm using [Training Demo With Installing FATE Only From Pypi] to test hetero-nn model on MULTI-CLASS datasets, but when using [loss = t.nn.CrossEntropyLoss()], the following error raised:

`0D or 1D target tensor expected, multi-target not supported`

It seems a MULTI-CLASS dataset is mis-considered as a MULTI-TARGET dataset, so what can i do to fix the issue？

我想只用 Pypi 来测试hetero-nn。但在使用多类别数据集和t.nn.CrossEntropyLoss()时却报错不支持“多任务”模型。也没有找到多分类问题example，我想知道是Pypi不支持吗？应该怎么做？

**Comment by talkingwallace** [comment]
这个是支持的 这个看来是dataset 返回label的格式问题，cross entropy需要一个1维的label tensor 看看修改下可不可以跑通？

**Comment by EthanLifeGreat** [comment]
> 这个是支持的 这个看来是dataset 返回label的格式问题，cross entropy需要一个1维的label tensor 看看修改下可不可以跑通？

![image](https://github.com/FederatedAI/FATE/assets/45615462/35fac1ca-16ec-4813-bd55-1da294994324)

label就是一维的，没做one-hot。然后top_model最后给出的是6个值（对应6个class），不知道这样是正确的吗？

**Comment by talkingwallace** [comment]
你可以在本地先试一下（不提交fate任务），model的输出能不能和label算出loss，按照现在看来是会报错的，可以在本地调一下的
